scripts/gulpfiles/appengine_tasks.js

```diff
@@ -53,6 +53,18 @@ function copyAppengineSrc() {
   return gulp.src(appengineSrc).pipe(gulp.dest(demoTmpDir));
 }
 
+/**
+ * Copies playground deps into deploy directory.
```

Comment:
This copy differs from the other `copyStaticSrc` and `copyAppengineSrc` since it copies uncommited local files. I think it's worth mentioning in the JsDoc about that (and perhaps mention that npm install needs to have been run).

If possible, I'd like for us to check that there is node_modules file somewhere in this step because otherwise it will just silently "fail" if someone was to have cleaned their node modules before running.

Reply:
Hmm, all good points. If the goal is to not have it depend on the current state of the repo, perhaps we can directly acquire the files we're interested in from unpkg.

Reply:
Also I noticed that the script reads from the local package.json to determine the version, rather than the one in HEAD. Should that also be switched over?

Reply:
The current deploy demo script uses files in HEAD, rather than local files because it was a direct translation of the steps we have done in the past. I want to note that we could consider changing it.
Assuming we're not:
- unpkg: +1 to using unpkg, assuming it's not too complicated (not necessary for this PR, but we could file an issue to add later)
- `package.json`: good catch. I think it probably would be better to use the version in HEAD

Reply:
In the interest of not letting this sit, I'm going to throw this in, and update the associated but with the remaining tasks.